pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = ["setuptools", "tomli", "wheel"]
```

Comment:
`wheel` should be unnecessary to specify in the current day, and I don't see why you're including `tomli`.

Reply:
tomli was for vendored versioneer to load config from `pyproject.toml`

Reply:
Well, it wasn't working, but [build-time dependency mode](https://github.com/python-versioneer/python-versioneer?tab=readme-ov-file#build-time-dependency-mode) did work.